### PR TITLE
[MIG] attachment_azure: Backport from 14.0 to 13.0

### PR DESCRIPTION
**Pull request by p-tombez**

### CODE REVIEW

**Comment by max3903** [comment]
@hiren-serpentcs You may want to review this...

File: attachment_azure/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2016-2019 Camptocamp SA
```
Comment (leemannd) [inline attachment_azure/__manifest__.py]: Could be updated

File: attachment_azure/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Knowledge Management",
+    "depends": ["base_attachment_object_storage"],
+    "external_dependencies": {
+        "python": ["azure-storage-blob", "azure-identity"],
+    },
+    "website": "https://github.com/camptocamp/odoo-cloud-platform",
+    "installable": True,
+    "development_status": "Beta",
```
Comment (leemannd) [inline attachment_azure/__manifest__.py:20]: Is it still in beta?

File: attachment_azure/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Copyright 2016-2019 Camptocamp SA
+# Copyright 2021 Open Source Integrators
```
Comment (leemannd) [inline attachment_azure/models/__init__.py]: Copyright could be removed from init files

File: attachment_azure/__manifest__.py (hunk 1 above)
Comment (yvaucher) [reply attachment_azure/__manifest__.py]: According to the change @p-tombez did to migrate it (only changing the version number) I'm more at ease with letting this at 2016-2019 for Camptocamp as OSI did all the work on version 14.0.

If we improve it in the future we will update the copyright.

**Review by yvaucher — APPROVED** [review]

File: attachment_azure/__manifest__.py (hunk 1 above)
Comment (yvaucher) [reply attachment_azure/__manifest__.py]: Nevermind, I missed the https://github.com/camptocamp/odoo-cloud-platform/pull/243/commits/223a84d6a99a05480b389bc88065003132fbebaf commit done by @vrenaville

**Review by leemannd — APPROVED** [review]

**Review by leemannd — APPROVED** [review]

**Pull request merged**